Data/ByteString.hs

```diff
@@ -889,6 +891,22 @@ take n ps@(BS x l)
     | otherwise = BS x n
 {-# INLINE take #-}
 
+-- | /O(1)/ 'takeEnd' @n xs@ is equivalent to @'drop' ('length' xs - n) xs@.
+-- Takes 'n' elements from end of bytestring.
```

Comment:
Please change `'n'` to `@n@`, otherwise Haddock complains that it is out of scope, because it tries to hyperlink it with its definition. [`'...'` syntax](https://haskell-haddock.readthedocs.io/en/latest/markup.html#hyperlinked-identifiers) is for publicly exposed entities, [`@...@`](https://haskell-haddock.readthedocs.io/en/latest/markup.html#emphasis-bold-and-monospaced-text) is just to use monospaced font.

Same for `dropEnd`.

https://travis-ci.org/github/haskell/bytestring/jobs/732483942#L351

Reply:
thanks didn't know that was the case, also didn't run haddock locally to see the warning do that's my bad sorry!